scss/global.scss

```diff
@@ -1,63 +1,88 @@
 html,
 body {
-    height: 100vh;
+	height: 100vh;
 }
 
 body {
-    background: #333;
+	background: #333;
 }
 
 body.light-theme {
-    background: #fff;
+	background: #fff;
 }
 
 .form-control-feedback {
-    color: #f00;
+	color: #f00;
 }
 
 .react-autosuggest__container {
-    position: relative;
+	position: relative;
 }
 
 .react-autosuggest__input--focused {
-    outline: none;
+	outline: none;
 }
 
 .react-autosuggest__input--open {
-    border-bottom-left-radius: 0;
-    border-bottom-right-radius: 0;
+	border-bottom-left-radius: 0;
+	border-bottom-right-radius: 0;
 }
 
 .react-autosuggest__suggestions-container {
-    display: none;
+	display: none;
 }
 
 .react-autosuggest__suggestions-container--open {
-    display: block;
-    position: absolute;
-    border: 1px solid #aaa;
-    border-bottom-left-radius: 4px;
-    border-bottom-right-radius: 4px;
-    z-index: 2;
-    color: $gray-800;
-    background: $white;
-    width: 100%;
-    overflow: auto;
-    max-height: 200px;
-    z-index: 2;
+	display: block;
+	position: absolute;
+	border: 1px solid #aaa;
+	border-bottom-left-radius: 4px;
+	border-bottom-right-radius: 4px;
+	z-index: 2;
+	color: $gray-800;
+	background: $white;
+	width: 100%;
+	overflow: auto;
+	max-height: 200px;
+	z-index: 2;
 }
 
 .react-autosuggest__suggestions-list {
-    margin: 0;
-    padding: 0;
-    list-style-type: none;
+	margin: 0;
+	padding: 0;
+	list-style-type: none;
 }
 
 .react-autosuggest__suggestion {
-    cursor: pointer;
-    padding: 10px 20px;
+	cursor: pointer;
+	padding: 10px 20px;
 }
 
 .react-autosuggest__suggestion--highlighted {
-    background-color: #ddd;
+	background-color: #ddd;
+}
+.container-github-button {
```

Comment:
You shouldn't need these style changes - you can use existing button styles. Check out http://www.rpthreadtracker.com/tools/extensions for an example of an external link with button styles - the code for it is in `src\display\views\tools\components\BrowserExtensionsPane.js`.

Reply:
Noted and on it rn. Thank you for pointing this out!